src/TBuffer.cpp

```diff
@@ -3385,18 +3385,19 @@ void TBuffer::appendBuffer(const TBuffer& chunk)
     append(QString(QChar::LineFeed), 0, 1, Qt::black, Qt::black, TChar::None);
 }
 
-int TBuffer::calcWrapPos(int line, int begin, int end)
+int TBuffer::calculateWrapPosition(int lineNumber, int begin, int end)
 {
-    const QString lineBreaks = ",.- \n";
-    if (lineBuffer.size() < line) {
+    const QString lineBreaks = QStringLiteral("- \n");
```

Comment:
This is the only relevant change - the rest was improving code readability while I dug down to the issue at hand.